package.json

```diff
@@ -21,6 +22,9 @@
     "p-map": "^4.0.0",
     "tldjs": "^2.3.1"
   },
+  "peerDependencies": {
+    "nodemon": "*"
```

Comment:
Wasn't sure wether this should become a dev or peer dependency. I decided for the latter because `nodemon` is most likely installed on the machine already (if you're a Node.js developer)